### DISCUSSION TITLE
NextCloud Server Encryption Key

### DISCUSSION THREAD

**Discussion by 32mgKoffein** [saber-notes/saber#433] [Q&A] [answered]
After trying for some time, I could not figure out where to find the encryption key needed to login into Nextcloud. Somewhere its stated that its randomly generated and stored on the device. As I said I cant find where this should be. There also seems to be no tutorial on how to log in into Nextcloud.

**Comment by adil192** [comment, marked as answer]
You need to pick an encryption password when you first login to Saber. The encryption key is different and is handled internally.

**Reply by adil192** [reply]
Following an email, I'd like to clarify that
- You choose the encryption password when you first login to the app, not the website.
- When signing up to the server website, there'll be no mention of the encryption password anywhere. This is because the encryption password is handled in-app only (intentionally).

**Reply by GSTbyMM** [reply]
Is there any way to reset the encryption password?

**Reply by QubaB** [reply]
To change enxcryption password you should delete Saber folder on nextcloud server and create your synchronization from scratch. 

It is described directly in nextcloud setup in Saber : 
0. Make sure syncing is complete (see the sync progress on the home screen).
1. Log out of Saber.
2. Go to your server website and delete your 'Saber' folder. This will delete all your notes from the server.
3. Log back in to Saber. You can choose a new encryption password when logging in.
4. Don't forget to log out and log back in to Saber on your other devices too.
